lib/dal/src/attribute/prototype/debug.rs

```diff
@@ -154,7 +156,9 @@ impl AttributePrototypeDebugView {
                 let values_for_arg = match value_source {
                     ValueSource::Secret(secret_id) => {
                         vec![FuncArgDebugView {
-                            value: Secret::payload_for_prototype_debug_view(secret_id)?,
+                            value: serde_json::to_value(format!(
+                                "[REDACTED KEY FOR SECRET (ID: {secret_id})]"
```

Comment:
Where all is the `AttributePrototypeDebugView` used? Putting on my Security Engineer Hat makes me want to say that this should be `[REDACTED]` without the `secret_id` so it can't expose that the same value is used in however many places (and exactly what those places are).

Reply:
It is used in the frontend to debug what values are used. I believe knowing the `SecretId` is fine because it refers to the `Secrets` available in the `Secret` modal and picker in the frontend.

Reply:
However, I am willing to drop this too. It has been mostly used for development purposes.

Reply:
Yeah, given that we expose which secret is used (not the value itself, but which secret), and exactly where each is used as part of the normal UI, it should be fine.

:removes tin-foil hat: